src/constants.ts

```diff
@@ -57,4 +59,14 @@ export const getRedirectPage = (): HTMLElement => {
   containerEl.appendChild(caption);
 
   return containerEl;
-};
\ No newline at end of file
+};
+
+export const WALLY_ROUTES: Record<WallyMethodName, string> = {
+  [WallyMethodName.ACCOUNTS]: 'me',
+  [WallyMethodName.REQUEST_ACCOUNTS]: 'me',
+  [WallyMethodName.SIGN]: 'wallet/sign-message',
+  [WallyMethodName.PERSONAL_SIGN]: 'wallet/sign-message',
```

Comment:
some of these endpoints still need to be written right?

Reply:
nah man we're all set 🐴 
https://github.com/Wally-xyz/wallet-api/pull/221

Reply:
unless you're specifically talking about `eth_sign` vs `personal_sign` - still not sure about the crypographic differences there - they just go to that same api endpoint, i'll add a note.

Reply:
nah i just meant do they exist at all. cool, forgot we added that